scripts/dabl_deploy.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2019, Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+
+
+# Terminate on any failure
+set -ex
+
+STAMP=$(date "+%H%M%S")
+PROJECT_NAME=Cbdc$STAMP
+
+LEDGER_NAME=cbdc
```

Comment:
just wondering: shouldn't the ledger name be unique as project name?

Reply:
I think it does not make a difference. I mean, I deployed 10-20 projects with ledgers (having the same name) after each other and they did not bother each other.